app/Jobs/GenerateHashValue.php

```diff
@@ -75,61 +75,62 @@ public function __construct(Image $image)
      * Execute the job.
      *
      * @return void
+     * @throws Exception
      */
     public function handle()
     {
-        try {
-            if ($this->needsHashValue($this->image)) {
-                FileCache::getOnce($this->image, function ($image, $path) {
-                    if (!File::exists($path)) {
-                        throw new Exception("File '{$path}' does not exist.");
-                    }
-                    if ($this->needsHashValue($image)) {
-                        $this->getThumbnail($image, $path);
-                        // here path or the image which i get with get Thumbnail?
-                        $output = $this->python("{$path}");
-                        $image->hash = $output;
-                        $image->save();
-                    }
-
-                });
-            }
-        } catch (Exception $e) {
-            if (App::runningUnitTests()) {
-                throw $e;
-            } else {
-                Log::warning("Could not process new image {$this->image->id}: {$e->getMessage()}");
-            }
-        }
+        $hashValue = $this->getHashValue($this->image);
+        $id = $hashValue->id;
+        $hash = $hashValue->hash;
+
+        dd($hash);
+        $this->image->hash = $hash;
```

Comment:
I am not sure if this is the correct syntax?
@mzur

Reply:
I'm not sure what `$hashValue->id` should mean. But to store the hash you do:
```php
$hash = $hashValue->hash;
$this->image->hash = $hash;
$this->image->save()
```